employee-rostering-frontend/src/store/contract/contract.test.ts

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { mockStore } from '../mockStore';
+import { AppState } from '../types';
+import * as actions from './actions';
+import reducer, { contractOperations } from './index';
+import {withElement, withoutElement, withUpdatedElement} from 'util/ImmutableCollectionOperations';
+import {onGet, onPost, onDelete} from 'store/rest/RestServiceClient';
```

Comment:
TypeScript compiler complains that
```
TS2305: Module '"../rest/RestServiceClient"' has no exported member 'onGet'.
```
Can you fix it? It seems that simply importing from `store/rest/__mocks__/RestServiceClient` fixes it. It's the easiest solution. Not sure if the correct and best one but could be.

Reply:
Changing it to that breaks the test. I am guessing the errors in the test are due to the TypeScript compiler in the IDE not being aware that the classes are being mocked. So we might need to change it in a compiler setting, or move the methods to a new file `RestTestUtils` (which would be strongly coupled with store/rest/__mocks__/RestServiceClient).